lib/usb_descriptors.c

```diff
@@ -71,25 +76,44 @@ uint8_t const* tud_descriptor_device_cb(void) {
 // Configuration Descriptor
 //--------------------------------------------------------------------+
 
-enum { ITF_NUM_MIDI = 0, ITF_NUM_MIDI_STREAMING, ITF_NUM_TOTAL };
+// enum { ITF_NUM_MIDI = 0, ITF_NUM_MIDI_STREAMING, ITF_NUM_TOTAL };
 
-#define CONFIG_TOTAL_LEN (TUD_CONFIG_DESC_LEN + TUD_MIDI_DESC_LEN)
+enum
+    {
+        ITF_NUM_CDC            = 0,
+        ITF_NUM_CDC_DATA,
+        ITF_NUM_MIDI,
+        ITF_NUM_MIDI_STREAMING,
+        ITF_NUM_TOTAL
+    };
 
-#if CFG_TUSB_MCU == OPT_MCU_LPC175X_6X || \
-    CFG_TUSB_MCU == OPT_MCU_LPC177X_8X || CFG_TUSB_MCU == OPT_MCU_LPC40XX
-// LPC 17xx and 40xx endpoint type (bulk/interrupt/iso) are fixed by its number
-// 0 control, 1 In, 2 Bulk, 3 Iso, 4 In etc ...
-#define EPNUM_MIDI 0x02
-#else
-#define EPNUM_MIDI 0x01
-#endif
+#define CONFIG_TOTAL_LEN (TUD_CONFIG_DESC_LEN + TUD_CDC_DESC_LEN + TUD_MIDI_DESC_LEN)
+
+/* #if CFG_TUSB_MCU == OPT_MCU_LPC175X_6X ||                               \
+ *     CFG_TUSB_MCU == OPT_MCU_LPC177X_8X || CFG_TUSB_MCU == OPT_MCU_LPC40XX
+ * // LPC 17xx and 40xx endpoint type (bulk/interrupt/iso) are fixed by its number
+ * // 0 control, 1 In, 2 Bulk, 3 Iso, 4 In etc ...
+ * #define EPNUM_MIDI 0x02
+ * #else
+ * #define EPNUM_MIDI 0x01
+ * #endif */
+
+#define EPNUM_MIDI 0x04
```

Comment:
`EPNUM_MIDI` needs to not conflict any of the `EPNUM_CDC_*` (and it would w/ `0x80 | EPNUM_MIDI`). so i changed it to 4.

i'm not sure what the if else based on the MCU type did. i thought you could maybe set those arbitrarily.?